### ISSUE TITLE
The upstream Firefox/Gecko version should also be shown in the About window

### ISSUE THREAD

**Issue by akash07k** [BrowserWorks/waterfox#3310]
**Is your feature request related to a problem? Please describe.**
Currently, the upstreame Firefox/Gecko version does not show in the About Window of the Waterfox.

**Describe the solution you'd like**
The upstream Firefox/Gecko version should also be shown in the About window
**Describe alternatives you've considered**

**Additional context**

**Comment by sander-nl** [comment]
I agree. It helps with knowing which extensions are compatible. Also, for my work as an auditor, I need to report the browser versions I'm using and I'd like to replace Firefox with Waterfox.
